packages/ncids-css/packages/nci-card/internal/_nci-card.scss

```diff
@@ -1,9 +1,33 @@
 @use 'uswds-core' as *;
+@use "sass:meta" as meta;
 @use '../../../internal-uswds/usa-card/src/styles/usa-card' as card;
 
+// Mixin to pass for overriding certain external icon styles
+@mixin external-icon() {
```

Comment:
can you make these mixins something like "nci-card-external-icon" with the component name? Just so we don't get confused...

Reply:
I thought it being a mixin only used inside `_nci-card.scss` made that pretty obvious, but sure, whatever's best.